crates/tabby-common/src/config.rs

```diff
@@ -111,14 +111,21 @@ impl RepositoryConfig {
     }
 
     pub fn name(&self) -> String {
-        if let Some(name) = &self.name {
-            name.clone()
-        } else {
-            filenamify(&self.git_url)
-        }
+        self.name
+            .clone()
+            .unwrap_or_else(|| clean_name(filenamify(&self.git_url)))
     }
 }
 
+fn clean_name(s: String) -> String {
```

Comment:
```suggestion
fn sanitize_name(s: String) -> String {
```

add unit test for this? at least include `@` to test.